include/linker/intlist.ld

```diff
@@ -44,3 +45,18 @@ SECTION_PROLOGUE(.intList,,)
 	KEEP(*(.intList))
 }
 #endif
+#elif defined(__MWDT_LINKER_CMD__)
+#ifndef LINKER_PASS2
+SECTION_PROLOGUE(.intList,,)
+{
+	KEEP(*(.irq_info.*))
```

Comment:
It's cleaner to just add .irq_info.* to the pre-existing list of input sections than to ifdef the whole thing.

It will be harmless to gcc.

But a comment explaining why it's .irq_info. on MWDT in the commit message is in order.

Reply:
Agreed.  Just in general this patch has a TON of cut/paste linker code intended to work around what look to me like fairly trivial syntax changes between GNU binutils and MWDT.  Surely we can find one syntax that works for both, and where we can't we should probably be investigating our own tooling (e.g. python scripts run between linker passes, etc...) that can be made portable.

What we absolutely don't want is a bunch of custom code in our core infrastructure that can only be exercised by people using a proprietary toolchain for a single platform.  This code needs to run unmodified everywhere so that everyone has the ability to modify the linker framework.